openhands/runtime/impl/eventstream/eventstream_runtime.py

```diff
@@ -500,17 +497,14 @@ def run_action(self, action: Action) -> Observation:
                     self.log('debug', f'response: {response}')
                     error_message = response.text
                     self.log('error', f'Error from server: {error_message}')
-                    obs = FatalErrorObservation(
-                        f'Action execution failed: {error_message}'
+                    obs = ErrorObservation(
+                        f'Action execution failed: {error_message}',
```

Comment:
Just to note - this one is an ErrorObs for any non-200 status. Is it too broad? Are all those messages appropriate for the llm?

Reply:
ahh good catch. We actually raise for non-200, so this never gets hit. Removed